addons/mail/models/mail_render_mixin.py

```diff
@@ -332,6 +337,40 @@ def _render_template_qweb(self, template_src, model, res_ids,
 
         return results
 
+    @api.model
+    def _render_template_qweb_static(self, template_src, model, res_ids):
```

Comment:
Weird to call "static" something that is dynamic ^^ just a safer version of the other one.

maybe ``_render_template_qweb_regex`` (as I have no clear word for 'light', 'simple' in mind, maybe the implementation choice can be used as suffix).

or maybe ``_render_template_qweb_simple``